code/modules/clothing/suits/armor.dm

```diff
@@ -67,13 +67,22 @@
 /obj/item/clothing/suit/armor/hos/alt
 	name = "armored trenchoat"
 	desc = "A trenchcoat enchanced with a special lightweight kevlar. The epitome of tactical plainclothes."
-	icon_state = "hostrench"
-	item_state = "hostrench"
+	icon_state = "hostrench_open"
+	item_state = "hostrench_open"
 	flags_inv = 0
 	ignore_suitadjust = 0
+	suit_adjusted = 1
 	action_button_name = "Open/Close Trenchcoat"
 	adjust_flavour = "unbutton"
 
+/obj/item/clothing/suit/armor/hos/alt/verb/openjacket(var/mob/user)

```

Comment:
It's better to just have a single generic proc that has a semi-humorous message when you try to adjust something that is not adjustable than having the same exact thing copypasted repeatedly in several places.